Widgets/Animation/VarmintAnimation.cs

```diff
@@ -69,21 +71,21 @@ public void Reset()
         internal void Update(GameTime gameTime)
         {
             if (IsComplete) return;
-            _animationProgressSeconds += gameTime.ElapsedGameTime.TotalSeconds;
-            var delta = _animationProgressSeconds;
+            if (_animationStartTime == null) _animationStartTime = gameTime.TotalGameTime;
+            var animationProgressSeconds = (gameTime.TotalGameTime - _animationStartTime.Value).TotalSeconds;
             if (_animationDurationSeconds > 0)
             {
-                if (_animationProgressSeconds > _animationDurationSeconds)
+                Animate(animationProgressSeconds / _animationDurationSeconds);
```

Comment:
We should garantee to the caller that the animation will never be called with a value > 1.0.   This can generate values bigger than that.